src/App.tsx

```diff
@@ -1,24 +1,48 @@
 import React from 'react';
-import logo from './logo.svg';
+import { PrimaryButton } from './components/buttons/button-primary';
+import { SecondaryButton } from './components/buttons/button-secondary';
+import { CTAButton } from './components/buttons/button-cta';
 import './App.css';
+import { H1 } from './styles/typography/h1';
+import { H2 } from './styles/typography/h2';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <H1> Styleguide </H1>
+      <H2> Botões </H2>
+      <PrimaryButton hasIcon={false} disable={false} size="default" onClick={() => alert('hello')}>
+        Botao primario
+      </PrimaryButton>
+      <br></br>
+      <PrimaryButton hasIcon={false} disable={false} size="compact">
+        Botao primario
+      </PrimaryButton>
+      <br></br>
+      <PrimaryButton hasIcon={false} disable={true} size="default">
```

Comment:
nit: nos casos de true, vc não precisa passar o valor:

```suggestion
      <PrimaryButton disable size="default">
```